test/controllers/helpers/AccountSummaryHelperSpec.scala

```diff
@@ -372,8 +373,133 @@ class AccountSummaryHelperSpec extends ViewSpecBase with MockitoSugar with Scala
     "return the generic error message" in {
       reset(mockVatService)
       when(mockVatService.fetchVatModel(Matchers.any())(Matchers.any())).thenReturn(Future.successful(VatGenericError))
-      val view = accountSummaryHelper().getAccountSummaryView(VatGenericError)(fakeRequestWithEnrolments)
+      val view = accountSummaryHelper().getAccountSummaryView(VatGenericError, Nil)(fakeRequestWithEnrolments)
       view.toString must include("We can’t display your VAT information at the moment.")
     }
   }
+
+  "the user has enrolment for Vat Var that is activated" should {
```

Comment:
This test doesn't need to be here as the logic it was created to test is now in the VatVarPartial Builder